packages/react/src/_popover/Popover.tsx

```diff
@@ -69,8 +73,9 @@ export function usePopover<RT extends ReferenceType = ReferenceType>(
 		hiddenWithCSS = false,
 		isOpen,
 		matchReferenceWidth = false,
-		minHeight: minHeightOption,
-		maxHeight: maxHeightOption,
+		minHeight,
+		maxHeight,
+		noVerticalScroll,
```

Comment:
I'm never a fan of double negative variable names. How about something like `allowsVerticalScroll` which is `true` by default

Reply:
Maybe I went the wrong direction with this? Really what we're doing is setting the height of the popover to match the height of its content. No vertical scrolling is a byproduct. What do you think about `setHeightToContentHeight`? Or something along those lines? I think I originally went with `fixHeightAsContentHeight` but it wasn't well received.

Reply:
Hmm yeah, a simple `fixedHeight` sounds good. With a JSDoc explaining what it's fixed to.

- [ ] `fixedHeight` with JSDoc